receiver/hostmetricsreceiver/internal/scraper/processscraper/metadata.yaml

```diff
@@ -58,7 +58,18 @@ metrics:
       monotonic: true
     attributes: [state]
 
+  # produced when receiver.hostmetricsreceiver.emitIncorrectProcessMemoryMetrics is enabled
   process.memory.physical_usage:
+    enabled: true
+    description: "Deprecated: use `process.memory.usage` metric instead. The amount of physical memory in use."
+    unit: By
+    sum:
+      value_type: int
+      aggregation: cumulative
+      monotonic: false
+
+  # produced when receiver.hostmetricsreceiver.emitCorrectProcessMemoryMetrics is enabled
+  process.memory.usage:
     enabled: true
```

Comment:
This should be disabled in the first release. And it should be swapped with `process.memory.physical_usage` in the next iteration. We should never have both enabled by default.

Reply:
nvm. I thought it's already migrated from the feature gates.

Reply:
Yeah it was just a rebase with no changes